website/src/supportedNetworks/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { NetworksRegistry } from '@pinax/graph-networks-registry'
+import { useRouter } from 'next/router'
+import { useCallback, useMemo, useState } from 'react'
+
+import { ExperimentalLink } from '@edgeandnode/gds'
+
+import { Callout } from '@/components'
+import { useI18n } from '@/i18n'
+
+import { EmptySearchResults, NetworkFilters, NetworksTable } from './components'
+import type { NetworkData, ProcessedNetwork } from './utils'
+import { processNetworksData } from './utils'
+
+export async function getSupportedNetworks(): Promise<ProcessedNetwork[]> {
+  const registry = await NetworksRegistry.fromLatestVersion()
+  return processNetworksData(registry.networks as NetworkData[])
+}
+
+export async function getSupportedNetworksStaticProps() {
+  return {
+    props: {
+      networks: await getSupportedNetworks(),
+    },
+  }
+}
+
+export function SupportedNetworksTable({
+  networks,
+}: Awaited<ReturnType<typeof getSupportedNetworksStaticProps>>['props']) {
+  const { t } = useI18n()
+  const router = useRouter()
+  const { locale = 'en' } = router
+  const [searchQuery, setSearchQuery] = useState('')
+  const [showTestnets, setShowTestnets] = useState(false)
+
+  const filteredNetworks = useMemo(() => {
+    let filtered = networks
+
+    // Filter by testnet
+    if (!showTestnets) {
+      filtered = filtered.filter((network) => String(network.networkType) !== 'testnet')
+    }
+
+    // Filter by search
+    if (searchQuery) {
+      const query = searchQuery.toLowerCase()
+      filtered = filtered.filter(
+        (network) =>
+          network.id.toLowerCase().includes(query) ||
+          network.shortName.toLowerCase().includes(query) ||
+          network.fullName.toLowerCase().includes(query),
+      )
+    }
+
+    return filtered
+  }, [networks, searchQuery, showTestnets])
+
+  const noResults = filteredNetworks.length === 0 && searchQuery !== ''
+
+  const handleClearSearch = useCallback(() => {
+    setSearchQuery('')
+  }, [])
+
+  const handleShowTestnets = useCallback(() => {
+    if (noResults) {
+      setSearchQuery('')
+    }
+    setShowTestnets(true)
+  }, [noResults])
+
+  const handleHideTestnets = useCallback(() => {
+    setShowTestnets(false)
+  }, [])
+
+  const handleToggleTestnets = useCallback(() => {
+    if (!showTestnets) {
+      handleShowTestnets()
+    } else {
+      handleHideTestnets()
+    }
+  }, [showTestnets, handleShowTestnets, handleHideTestnets])
+
+  const setSearchQueryCallback = useCallback((value: string) => {
+    setSearchQuery(value)
+  }, [])
+
+  return (
+    <>
+      <Callout variant="info" className="mb-6 flex items-center">
+        <span className="text-p14">
+          {t('index.supportedNetworks.infoText')}{' '}
+          <ExperimentalLink href="https://thegraph.com/docs/en/indexing/chain-integration-overview/">
+            {t('index.supportedNetworks.infoLink')}
+          </ExperimentalLink>
+          .
+        </span>
+      </Callout>
```

Comment:
```suggestion
      <Callout variant="info" className="mb-6">
        {t('index.supportedNetworks.infoText')}{' '}
        <ExperimentalLink href="https://thegraph.com/docs/en/indexing/chain-integration-overview/">
          {t('index.supportedNetworks.infoLink')}
        </ExperimentalLink>
        .
      </Callout>
```
(as discussed in [this thread](https://github.com/graphprotocol/docs/pull/913#discussion_r2025049820))

Reply:
(Feel free to update `Callout` to fix the font size, sorry I can't make the suggestion here because that file hasn't been updated obviously.)

Reply:
Oh, and also, related to the separation of `infoText` and `infoLink`: you can actually use a single translation string for that (and I would also include the period at the end in it), and I believe it's better because some languages may want to translate the sentence in a way that the link is not at the very end. The way to do that is to use a `{0}` placeholder for the link, and pass an array containing the string or `ReactNode` to replace it with as the second argument to `t()`. Check out `index.supportedNetworks.description.base` (in this same codebase) to see what I mean!

Reply:
Though... I just remembered that `infoLink` is a self-contained CTA in this case, and not the continuation of a sentence like the code looks like. So I guess it's not super important here.